SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/ApplicationInsights.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
On my previous comment I mentioned that we should not include it if this is a Windows10 specific file. If its not then there is no harm in including it. 

So if you are using VS2015 on Windows 8.1, this solution will still work?